builtin/game/item.lua

```diff
@@ -1,4 +1,22 @@
--- Minetest: builtin/item.lua
+---------------------------------------------------------------------------------
```

Comment:
That not right. Only builtin/game/ has no headers yet, everywhere else are often some. In mainmenu 99% have.
I would have said that it is the state of 2013, but ok, i can change it to 2015. Also moving it out of this PR might be good.